tests/flow/test_cdlp.py

```
@@ -0,0 +1,503 @@
+
+                # unexpected extra parameters
+                """CALL algo.labelPropagation({nodeLabels: ['Person'],
+                               relationshipTypes: ['KNOWS'],
+                               invalidParam: 'value'})""",
+        ]
+
+        for q in invalid_queries:
+            try:
+                self.graph.query(q)
+                self.env.assertFalse(True)
+            except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)